Directory.Build.props

```diff
@@ -0,0 +1,77 @@
+<Project>
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <TargetFramework Condition="'$(Mono)' == ''">net5.0</TargetFramework>
```

Comment:
It will cause a unsupported behavior on `OpenRA.Test` if running with Mono, but it will only be a problem if someone tries to build it directly as the test project is only built directly and not in the sln.